src/api/tigl.h

```diff
@@ -3022,6 +3022,49 @@ TIGL_COMMON_EXPORT TiglReturnCode tiglIntersectWithPlane(TiglCPACSConfigurationH
                                                          char** intersectionID);
 
 /**
+* @brief tiglIntersectWithPlaneSegment computes the intersection line(s) between a shape
+* and a plane segment. The plane segment is defined by p(u,v) = p1 (1-u) + p2u + wv, with
+* u in [0,1]. Ideally w should be perpendicular to (p1 - p2)
+*
+* It returns an intersection ID for further computations on the result.
+* To query points on the intersection line, ::tiglIntersectGetPoint has
+* to be called.
+*
+* The shape has to be specified by its CPACS UID.
+* The plane segment is specified by two points p1 and p2  and a normal vector w, which
+* (ideally) is perpendicular to (p2 - p1). The normal vector w must not be zero!
+*
```

Comment:
Please add a sketch (image) to the function, such that the functioning becomes more clear for the user.

Have a look at the documentation of the function ```tiglWingComponentSegmentPointGetSegmentEtaXsi```